app/javascript/App.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Home from './pages/Home';
+import About from './pages/about';
+import HotelList from './pages/HotelList';
+import ReservationList from './pages/ReservationList';
+import ReservedDetails from './pages/Reserve/ReservedDetails';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './App.css'
+import CreateUser from './components/CreateLogin';
+import UserLogin from './components/UserLogin';
+import BookReservations from './components/BookReservations';
+
+  
+function App() {
```

Comment:
- [ ] Please, kindly refactor this functional component  to ES6 arrow function, so your code complies with the latests standards.

The function keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non best practice.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can make a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.